ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -158,7 +158,7 @@ export const researchPurposeQuestions = [
     description: <div>If you are exploring the data at this stage to formalize a specific
       research question, please describe the reason for exploring the data, and the scientific
       question you hope to be able to answer using the data. <br/>
-      (Free text; 500 Character limit)</div>
+      (Free text; 1000 Character limit)</div>
```

Comment:
I would explicitly say 50-1000 characters if we're blocking on < 50.

Reply:
Now that we not blocking on 50 so i'll keep it as it as 1000